.changelog/1170.txt

```diff
@@ -0,0 +1,7 @@
+```release-note:note
+dns: clean up UpdateDNSRecordParams and skip record lookup
+```
+
+```release-note:breaking-change
+dns: many fields have been removed from UpdateDNSRecordParams
```

Comment:
we need to be specific here what is removed for those looking through the CHANGELOG

Reply:
Done.